reactor-core/src/test/java/reactor/core/publisher/FluxRepeatWhenTest.java

```diff
@@ -512,5 +458,7 @@ public void repeatWhenContextTrigger_OriginalContextManuallyUpdated() {
 				"emitted 2 elements this attempt, 0 repeats left");
 
 		assertThat(contexts).allMatch(ctx -> ctx.hasKey("thirdPartyContext"));
+
+		System.out.println(contexts);
```

Comment:
leftover debug code?

Reply:
it was there before, just GitHub renders it as an addition. Can remove 👍